File: android/app/src/main/java/com/lightningkite/kwifttemplate/shared/views/ExampleContent2ViewData.kt
package com.lightningkite.kwifttemplate.shared.views

import com.lightningkite.kwift.actuals.SecurePreferences
import com.lightningkite.kwift.shared.ObservableProperty
import com.lightningkite.kwift.shared.ViewData
import com.lightningkite.kwift.shared.ViewDataStack
import com.lightningkite.kwifttemplate.shared.models.*

class ExampleContent2ViewData(val stack: ViewDataStack) : ViewData {
    override val title: String get() = "Example Content 2"
}

File: ios/com/lightningkite/kwifttemplate/shared/views/ExampleContent2ViewData.swift
//Package: com.lightningkite.kwifttemplate.shared.views
//Converted using Kwift2

import Foundation



public class ExampleContent2ViewData: ViewGenerator {
    
    public var stack: ObservableStack<ViewGenerator>
    
    override public var title: String {
        get {
            return "Example Content 2 "
        }
    }
    
    override public func generate(dependency: ViewDependency) -> View {
        var xml = ExampleContent2Xml()
        var view = xml.setup(dependency)
        return view
    }
    override public func generate(_ dependency: ViewDependency) -> View {
        return generate(dependency: dependency)
    }
    
    public init(stack: ObservableStack<ViewGenerator>) {
        self.stack = stack
        super.init()
    }
    convenience public init(_ stack: ObservableStack<ViewGenerator>) {
        self.init(stack: stack)
    }
}
 
